### ISSUE TITLE
flutter/flutter CI should automatically detect all dart packages under //dev and run their tests

### ISSUE THREAD

**Issue by christopherfujino** [flutter/flutter#136500]
Today, tests under //dev are run as part of the framework misc shard, specified by this code: https://github.com/flutter/flutter/blob/3f722c1442fa07e6afc2442853bc2a7813a3ba8c/dev/bots/test.dart#L982.

This requires hard coding the path to each dart package and then invoking `flutter test` or `dart test` on them. However, it is easy to overlook this (for example, currently tests under //dev/tools do not get run). Instead, test.dart should automatically detect every pubspec.yaml under //dev and run flutter test on it.

**Comment by Hixie** [comment]
@christopherfujino I agree that this would be valuable. I'm not sure having an issue to track it is going to help though (anyone can do this, and there's no particular owner for this logic). If nobody is running to implement this, should we close this issue for now?

**Comment by christopherfujino** [comment]
> @christopherfujino I agree that this would be valuable. I'm not sure having an issue to track it is going to help though (anyone can do this, and there's no particular owner for this logic). If nobody is running to implement this, should we close this issue for now?

i won't object to closing this. could you clarify our policy for which bugs should be open and which closed? should those with no owner and no assignment be closed?

**Comment by Hixie** [comment]
https://github.com/flutter/flutter/wiki/Issue-hygiene#closing-issues is the answer to the above question, such as it is

**Comment by Al7mza** [comment]
Hi bro, I extracted a texture from the asset file for the purpose of modifying it (it took me three months to find and modify it) and now I want to add the modification to the asset file. Can you help me
🥲🙏🙏🙏
